skhep/dataset/selection.py

```diff
@@ -202,8 +218,8 @@ def rootselection(self):
         Return the selection readable for RootDataset
         """
         
-        selection = self._selection.replace("&","&&")
-        selection = selection.replace("|","||")
+        selection = self._selection.replace(" & "," && ")
```

Comment:
Should one add a comment that the formulate package might be a good thing to exploit here in the future?